manifests/py2wasm/py2wasm.json

```diff
@@ -1,40 +1,40 @@
 {
   "name": "py2wasm",
-  "description": "A plugin to convert Python applications to Spin compatible modules",
+  "description": "A plugin to convert py files to Spin compatible modules",
```

Comment:
Why did this get changed back to the old wording?

Reply:
Ah, it comes from my template. good catch. I will fix that.

Reply:
Fixed it now.